lib/registrations/registration_checker.rb

```diff
@@ -12,13 +12,16 @@ def self.apply_payload(registration, raw_payload)
         competing_payload = raw_payload['competing']
         comment = competing_payload&.dig('comment')
         organizer_comment = competing_payload&.dig('organizer_comment')
+        competing_status = competing_payload&.dig('status')
 
         new_registration.comments = comment if competing_payload&.key?('comment')
         new_registration.administrative_notes = organizer_comment if competing_payload&.key?('organizer_comment')
+        new_registration.competing_status = competing_status if competing_payload&.key?('status')
```

Comment:
Could we not just do `competing_status.present?`

Reply:
In this specific case, we could. But in general we can't, because for some fields (like organizer comments for example, where the organizer accidentally entered something wrong and wants to undo the comment) we want to allow deliberately setting the value to `nil`. But `nil.present?` returns false, so the explicitly, intently passed `nil` would never be written.
In the context of competing_status specifically, `nil` doesn't make sense but for the sake of consistency we use `key?` like everywhere else.